packages/api/src/beacon/routes/beacon/block.ts

```diff
@@ -229,8 +229,12 @@ export type Api = {
    * Retrieves BlobSidecar included in requested block.
    * @param blockId Block identifier.
    * Can be one of: "head" (canonical head in node's view), "genesis", "finalized", \<slot\>, \<hex encoded blockRoot with 0x prefix\>.
+   * @param indices Array of indices for blob sidecars to request for in the specified block. Returns all blob sidecars in the block if not specified.
    */
-  getBlobSidecars(blockId: BlockId): Promise<
+  getBlobSidecars(
+    blockId: BlockId,
+    indices?: string[]
```

Comment:
Indices can only be numbers, we can handle that the same way as epoch and slot

```suggestion
    indices?: number[]
```